client.go

```diff
@@ -26,6 +27,8 @@ type Conn struct {
 type Client struct {
 	Net            string            // if "tcp" a TCP query will be initiated, otherwise an UDP one (default is "" for UDP)
 	UDPSize        uint16            // minimum receive buffer for UDP messages
+	TLS            bool              // enables TLS connection (port 853)
```

Comment:
@andrewtj and actual flag.FlagSet? Or a ClientOpt{} struct that can be filled with options? Or a plain string that we overload, just like Network?